src/message/MessageContainer.js

```diff
@@ -104,7 +104,7 @@ class MessageContainer extends PureComponent {
         twentyFourHourTime={twentyFourHourTime}
         ownEmail={auth.email}
         onLongPress={this.handleLongPress}
-        starred={this.isStarred(message)}
+        starred={message.isOutbox ? undefined : this.isStarred(message)}
```

Comment:
But also, likely does not need to be handled at all.

As this will always return false:
```js
isStarred(message: Object) {
    const { flags } = this.props;
    return message.id in flags.starred;
  }
```